bleedingEdge.neon

```diff
@@ -2,6 +2,8 @@ parameters:
 	drupal:
 		bleedingEdge:
 			checkDeprecatedHooksInApiFiles: true
+			checkCoreDeprecatedHooksInApiFiles: false
+			checkContribDeprecatedHooksInApiFiles: false
```

Comment:
In bleedingEdge these should be `true`, as it turns on all feature flags

Reply:
Ok